yt_dlp/extractor/sovietscloset.py

```diff
@@ -0,0 +1,217 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import datetime
+
+from ..utils import js_to_json
+from .common import InfoExtractor
+
+
+class SovietsClosetExtractor(InfoExtractor):
+    MEDIADELIVERY_REFERER = {'Referer': 'https://iframe.mediadelivery.net/'}
+
+    def parse_nuxt_jsonp(self, nuxt_jsonp_url, video_id, name):
+        NUXT_JSONP_RE = r'__NUXT_JSONP__\(.*?\(function\((?P<arg_keys>.*?)\)\{return\s(?P<js>\{.*?\})\}\((?P<arg_vals>.*?)\)'
+        nuxt_jsonp = self._download_webpage(nuxt_jsonp_url, video_id, note=f'Downloading {name} __NUXT_JSONP__')
+
+        js, arg_keys, arg_vals = self._search_regex(NUXT_JSONP_RE, nuxt_jsonp, '__NUXT_JSONP__', group=['js', 'arg_keys', 'arg_vals'])
+
+        arg_keys = arg_keys.split(',')
+        arg_vals = arg_vals.split(',')
+        args = {key: val for key, val in zip(arg_keys, arg_vals)}
+
+        for key, val in args.items():
+            if val in ('undefined', 'void 0'):
+                args[key] = 'null'
+
+        return self._parse_json(js_to_json(js, args), video_id)['data'][0]
+
+    def video_meta(self, video_id, stream_date, game_name, catgory_name, episode_number):
+        title = f'SovietWomble {stream_date} - {game_name}'
+        if catgory_name != 'Misc':
+            title += f' - {catgory_name}'
```

Comment:
```suggestion
    def video_meta(self, video_id, stream_date, game_name, category_name, episode_number):
        title = f'SovietWomble {stream_date} - {game_name}'
        if category_name != 'Misc':
            title += f' - {category_name}'
```

Reply:
Done in https://github.com/yt-dlp/yt-dlp/pull/884/commits/5b23de60ce359763b8e6f2258150b7921945add8.